2023-07-18 15:27:29,665 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Current SDK version is 0.15.5
2023-07-18 15:27:29,665 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Configure stats pid to 2968
2023-07-18 15:27:29,665 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Loading settings from /Users/heroofwisdom/.config/wandb/settings
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Loading settings from /Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/wandb/settings
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_init.py:_log_setup():507] Logging user logs to /Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/wandb/run-20230718_152729-zmhf7oa8/logs/debug.log
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/wandb/run-20230718_152729-zmhf7oa8/logs/debug-internal.log
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_init.py:init():547] calling init triggers
2023-07-18 15:27:29,666 INFO    MainThread:2968 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-07-18 15:27:29,667 INFO    MainThread:2968 [wandb_init.py:init():571] re-initializing run, found existing run on stack: aqp7jljx
2023-07-18 15:27:29,669 INFO    MainThread:2968 [wandb_run.py:_finish():1887] finishing run alanfiler/90A_10B Multilabel Stratified/aqp7jljx
2023-07-18 15:27:29,669 INFO    MainThread:2968 [jupyter.py:save_history():445] not saving jupyter history
2023-07-18 15:27:29,670 INFO    MainThread:2968 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2023-07-18 15:27:29,670 INFO    MainThread:2968 [wandb_init.py:_jupyter_teardown():435] cleaning up jupyter logic
2023-07-18 15:27:29,670 INFO    MainThread:2968 [wandb_run.py:_atexit_cleanup():2121] got exitcode: 0
2023-07-18 15:27:29,670 INFO    MainThread:2968 [wandb_run.py:_restore():2104] restore
2023-07-18 15:27:29,670 INFO    MainThread:2968 [wandb_run.py:_restore():2110] restore done
2023-07-18 15:28:00,875 INFO    MainThread:2968 [wandb_init.py:init():596] starting backend
2023-07-18 15:28:00,876 INFO    MainThread:2968 [wandb_init.py:init():600] setting up manager
2023-07-18 15:28:00,878 INFO    MainThread:2968 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-07-18 15:28:00,879 INFO    MainThread:2968 [wandb_init.py:init():606] backend started and connected
2023-07-18 15:28:00,886 INFO    MainThread:2968 [wandb_run.py:_label_probe_notebook():1233] probe notebook
2023-07-18 15:28:00,886 INFO    MainThread:2968 [wandb_run.py:_label_probe_notebook():1243] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-07-18 15:28:00,886 INFO    MainThread:2968 [wandb_init.py:init():705] updated telemetry
2023-07-18 15:28:00,914 INFO    MainThread:2968 [wandb_init.py:init():738] communicating run to backend with 60.0 second timeout
2023-07-18 15:28:14,369 WARNING MainThread:2968 [wandb_init.py:init():1176] interrupted
Traceback (most recent call last):
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1154, in init
    run = wi.init()
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 741, in init
    result = run_init_handle.wait(
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2023-07-18 15:38:53,101 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Current SDK version is 0.15.5
2023-07-18 15:38:53,101 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Configure stats pid to 2968
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Loading settings from /Users/heroofwisdom/.config/wandb/settings
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Loading settings from /Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/wandb/settings
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_init.py:_log_setup():507] Logging user logs to /Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/wandb/run-20230718_153853-4qaskgx4/logs/debug.log
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/wandb/run-20230718_153853-4qaskgx4/logs/debug-internal.log
2023-07-18 15:38:53,102 INFO    MainThread:2968 [wandb_init.py:_jupyter_setup():453] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x16147b010>
2023-07-18 15:38:53,103 INFO    MainThread:2968 [wandb_init.py:init():547] calling init triggers
2023-07-18 15:38:53,103 INFO    MainThread:2968 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-07-18 15:38:53,103 INFO    MainThread:2968 [wandb_init.py:init():596] starting backend
2023-07-18 15:38:53,103 INFO    MainThread:2968 [wandb_init.py:init():600] setting up manager
2023-07-18 15:38:53,105 INFO    MainThread:2968 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-07-18 15:38:53,106 INFO    MainThread:2968 [wandb_init.py:init():606] backend started and connected
2023-07-18 15:38:53,110 INFO    MainThread:2968 [wandb_run.py:_label_probe_notebook():1233] probe notebook
2023-07-18 15:38:53,111 INFO    MainThread:2968 [wandb_run.py:_label_probe_notebook():1243] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-07-18 15:38:53,111 INFO    MainThread:2968 [wandb_init.py:init():705] updated telemetry
2023-07-18 15:38:53,151 INFO    MainThread:2968 [wandb_init.py:init():738] communicating run to backend with 60.0 second timeout
2023-07-18 15:39:11,012 WARNING MainThread:2968 [wandb_init.py:init():1176] interrupted
Traceback (most recent call last):
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1154, in init
    run = wi.init()
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 741, in init
    result = run_init_handle.wait(
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/heroofwisdom/Documents/GitHub/Citrine_Test_V3/.venv/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
